### PR TITLE
[MOSIP-31033] Code Merge from develop to release branch.

### PR DESCRIPTION
**Pull request by mahammedtaheer**

### CODE REVIEW

**Review by LoganathanSekar7627 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-[![Maven Package upon a push](https://github.com/mosip/mosip-openid-bridge/actions/workflows/push_trigger.yml/badge.svg?branch=release-1.2.0.1)](https://github.com/mosip/mosip-openid-bridge/actions/workflows/push_trigger.yml)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=mosip_mosip-openid-bridge&metric=alert_status)](https://sonarcloud.io/dashboard?branch=release-1.2.0.1&id=mosip_mosip-openid-bridge)
+[![Maven Package upon a push](https://github.com/mosip/mosip-openid-bridge/actions/workflows/push_trigger.yml/badge.svg?branch=develop)](https://github.com/mosip/mosip-openid-bridge/actions/workflows/push_trigger.yml)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=mosip_mosip-openid-bridge&metric=alert_status)](https://sonarcloud.io/dashboard?branch=develop&id=mosip_mosip-openid-bridge)
```
Comment (ase-101) [inline README.md:2]: branch query param is pointing to "develop"

File: kernel/pom.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<version>1.2.0.1-B3</version>
+	<version>1.2.0.1-B3-SNAPSHOT</version>
```
Comment (ase-101) [inline kernel/pom.xml:5]: child pom's are pointing to B4-SNAPSHOT and parent pom its B3-SNAPSHOT. is this ok?

File: README.md (hunk 1 above)
Comment (mahammedtaheer) [reply README.md]: correct the version

File: kernel/pom.xml (hunk 2 above)
Comment (mahammedtaheer) [reply kernel/pom.xml]: corrected version

**Review by ckm007 — APPROVED** [review]

**Pull request merged**